dynawo/sources/Solvers/AlgebraicSolvers/DYNSolverKINAlgRestoration.h

```diff
@@ -34,7 +34,7 @@ class Model;
  * @brief class Solver KINSOL for algebraic equations restoration and initialization
  * Generic class for KINSOL solver
  */
-class SolverKINAlgRestoration : public SolverKINCommon, private boost::noncopyable{
```

Comment:
Mean that we have to define copy and assignment methods, why is it needed now?

Reply:
Private inheritance from boost::noncopyable has been defined at an upper level (DYNSolverKINCommon) so it is inherited in this one